posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -324,38 +287,32 @@ async def test_insert_into_s3_activity_puts_data_into_s3(
         ):
             await activity_environment.run(insert_into_s3_activity, insert_inputs)
 
-    await assert_events_in_s3(s3_client, bucket_name, prefix, events, compression, exclude_events)
+    await assert_events_in_s3(
+        minio_client,
+        bucket_name,
+        prefix,
+        events=events + events_to_exclude + events_with_no_properties,
```

Comment:
Eventually, we could test that those events are present when we do not exclude them, but that's not how the test is designed, and that looks more like a normal event check.

Reply:
So, yeah, not doing much I think

Reply:
I guess it's just nice to know that the events are there even though you are excluding them 😅

Reply:
Anyways, to avoid confusion, I'm not passing `events_to_exclude` anymore (and not even assigning them, just generating them).

Reply:
Eventually, we could extend the `generate_test_events_in_clickhouse` to support generating multiple event names at the same time and generate both events to export and to exclude simultaneously. This could be useful to also test for `include_events` 🤔

Reply:
I'll do a follow-up to tackle that 👍